source/ht_file.cpp

```diff
@@ -78,8 +78,7 @@ namespace Hatchit
         File::File(void)
             : IFile(),
             m_position(0),
-            m_size(0),
-            m_handle(nullptr) {}
+            m_size(0) {}
```

Comment:
The point still stands that we want to make sure everything is initialized before we use it. @Honeybunch we're putting that in the coding standards, correct?